tsconfig.json

```diff
@@ -10,10 +10,9 @@
     "useUnknownInCatchVariables": true,
     "incremental": true,
     "tsBuildInfoFile": "./tsconfig.tsbuildinfo",
-    "preserveSymlinks": true,
```

Comment:
Removing this ensures that when using Typescript/VS Code's Go to Definition it'll go to the original source file, and not the symlink path in node_modules.